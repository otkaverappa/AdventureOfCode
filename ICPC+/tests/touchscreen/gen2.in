19
lg 10
vz
yy
tn
pd
ry
gu
ap
tr
tw
fe
an 10
sf
wx
pp
mc
xh
bw
fx
jw
cc
vi
wv 10
xs
oq
ot
xq
gr
xd
om
uc
rr
ju
nq 10
tx
mv
zh
dy
fd
st
vg
nu
yv
lv
ab 10
ku
db
mo
xb
er
yr
ma
as
af
xs
zu 10
ao
py
jc
tm
gd
gj
gu
ye
we
xw
vl 10
ww
fy
dc
qc
zs
so
qe
sm
qy
ry
ky 10
vk
et
od
pm
oo
iu
ml
yf
qx
xk
nq 10
og
eg
gv
fq
vc
cb
vr
ek
dt
an
pn 10
aj
us
lr
dy
jt
gn
an
jh
fg
ji
ie 10
bm
qg
hr
tw
gv
ma
ox
tt
xd
mg
qm 10
vb
ta
ie
ks
kl
gb
tq
uo
ma
ky
cy 10
xx
tx
af
dt
uy
wp
yg
ul
ae
yh
hs 10
xb
gl
ds
mf
sj
dl
gf
rl
al
jz
ak 10
fw
xg
bv
pi
po
kw
zp
on
vh
xa
uf 10
hl
tk
yc
jb
oq
xl
wy
hn
jw
bv
ud 10
kl
qh
up
jo
vt
co
dc
sm
dj
cb
ub 10
bd
qk
cs
hy
vs
jl
bf
dl
wy
ey
oj 10
ai
xg
rb
ho
ci
rv
vv
nc
uk
uf
