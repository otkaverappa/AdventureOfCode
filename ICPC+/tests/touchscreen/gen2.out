pd 4
yy 6
gu 7
tn 7
tr 7
fe 8
ry 8
tw 9
vz 11
ap 14
xh 3
sf 4
cc 6
wx 6
fx 7
vi 8
mc 10
bw 11
jw 12
pp 16
rr 4
xd 4
xs 5
gr 6
uc 6
xq 7
om 10
ot 10
ju 11
oq 12
fd 6
mv 6
nu 6
tx 6
vg 7
yv 7
dy 9
lv 9
st 9
zh 11
af 2
db 2
xb 2
as 4
er 6
xs 6
yr 9
ku 11
ma 12
mo 13
ao 3
gu 5
gj 6
xw 6
we 7
tm 8
gd 10
ye 11
py 12
jc 13
so 4
fy 5
ry 6
sm 6
dc 9
qy 9
zs 10
qc 12
qe 12
ww 12
iu 2
oo 5
ml 6
od 6
pm 6
yf 6
et 7
vk 8
xk 10
qx 14
fq 3
vr 5
vc 6
gv 7
dt 8
cb 9
eg 10
og 10
an 13
ek 13
jh 5
gn 6
lr 6
jt 7
ji 8
us 8
fg 9
an 10
dy 10
aj 12
hr 4
ox 4
tw 4
tt 5
ma 6
mg 6
gv 7
xd 9
qg 10
bm 11
gb 7
vb 7
uo 10
kl 11
ky 11
ta 11
tq 12
ie 13
ks 14
ma 15
dt 2
ae 6
af 6
uy 6
yh 6
wp 7
xx 7
yg 7
tx 10
ul 10
ds 3
gf 3
jz 3
mf 4
gl 8
sj 9
xb 9
dl 10
rl 10
al 12
zp 4
xg 5
vh 6
xa 9
bv 10
fw 10
pi 11
on 12
po 12
kw 14
jb 3
yc 3
jw 4
bv 5
hn 5
oq 6
tk 6
hl 7
wy 8
xl 12
dc 6
jo 8
kl 8
up 8
vt 8
cb 9
dj 9
qh 9
sm 11
co 13
hy 5
bf 6
jl 6
bd 7
ey 7
wy 8
vs 9
cs 10
dl 10
qk 10
uk 3
uf 5
ho 7
rb 8
rv 9
ci 10
nc 10
ai 11
vv 11
xg 11
